### PR TITLE
Testing: Add Testcase_VNC_Graphics_Mode

### LINKED ISSUE #36: Testing: Rocky 9 does not provide Basic Graphics Mode installation option
Per the [RedHat Enterprise Linux 9 Release Notes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index) the Basic Graphics Mode installation option has been removed from the installer.

**The `basic graphics` mode has been removed from the boot menu**
Previously, the `basic graphics` mode was used to install RHEL on hardware with an unsupported graphics card or to work around issues in graphic drivers that prevented starting the graphical interface. With this update, the option to install in a `basic graphics` mode has been removed from the installer boot menu. Use the VNC installation options for graphical installations on unsupported hardware or to work around driver bugs.

For more information on installations using VNC, see the [Performing a remote RHEL installation using VNC](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/performing_a_standard_rhel_installation/index#performing_a_remote_rhel_installation_using_vnc) section.

([BZ#1961092](https://bugzilla.redhat.com/show_bug.cgi?id=1961092))

As a result it is proposed that the (Basic Graphics Mode behaviors)[https://wiki.rockylinux.org/team/testing/release_criteria/r9/9_release_criteria/#basic-graphics-mode-behaviors] requirement be removed from the Rocky 9 Release Critieria and replaced with a VNC Installation behaviors

### PR DESCRIPTION
**Pull request by tcooper**
This PR will add a testcase for r9 to document validation of VNC Graphics mode installations.

Resolves #36

### CODE REVIEW

File: docs/team/testing/QA/Testcase_VNC_Graphics_Mode.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Description
+This test case will verify that release-blocking installers function as intended using the VNC installation method on supported systems and classes of hardware.
+
+{% include 'testing/qa_testcase_supported_systems.md' %}
+
+## Setup
+1. Obtain access to supported system and hardware class to be installed.
+2. Prepare appropriate media for the selected ISO to be tested.
+    - Example: [QA:Testcase Media USB dd](Testcase_Media_USB_dd.md)
+3. Obtain access to remote system with TigerVNC installed to use for VNC connection.
```
Comment (lumarel) [inline docs/team/testing/QA/Testcase_VNC_Graphics_Mode.md:23]: I don't think it really has to be TigerVNC, just a VNC client, right?

Comment (tcooper) [reply docs/team/testing/QA/Testcase_VNC_Graphics_Mode.md:23]: Yes... I believe any VNC client will work. `tigervnc` and `vinagre` are the Rocky provided VNC clients. I'll reword the item suggest `tigervnc` and/or `vinagre` as being provided by Rocky but indicated that any VNC client should work.

File: docs/team/testing/QA/Testcase_Custom_Boot_Methods_Boot_Iso.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  prod: Rocky Linux
+  ver: 9
+  level: Final
+---
+
+!!! info "Associated release criterion"
+    This test case is associated with the [Release_Criteria#vnc-graphics-mode-behaviors](9_release_criteria.md#vnc-graphics-mode-behaviors) release criterion. If you are doing release validation testing, a failure of this test case may be a breach of that release criterion.
+
+## Description
+This is to verify that the Anaconda installer starts correctly when booting from the Rocky Linux boot.iso using a custom .
```
Comment (akatch) [inline docs/team/testing/QA/Testcase_Custom_Boot_Methods_Boot_Iso.md:15]: A custom what? I think you a word :)

**Review by akatch — CHANGES_REQUESTED** [review]
In addition to my one comment, I think you also need to add the VNC Graphics Mode page to qa_test_cases.md.

File: docs/team/testing/QA/Testcase_Custom_Boot_Methods_Boot_Iso.md (hunk 2 above)
Comment (tcooper) [reply docs/team/testing/QA/Testcase_Custom_Boot_Methods_Boot_Iso.md]: Thanks @akatch should be taken care of.

**Review by akatch — APPROVED** [review]

**Pull request merged**